src/Microsoft.DotNet.ProjectModel/RuntimeOutputFiles.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.IO;
+using NuGet.Frameworks;
+
+namespace Microsoft.DotNet.ProjectModel
+{
+    public class RuntimeOutputFiles : CompilationOutputFiles
+    {
+        public RuntimeOutputFiles(string basePath,
+            Project project,
+            string configuration,
+            NuGetFramework framework) : base(basePath, project, configuration, framework)
+        {
+        }
+
+        public string Executable
```

Comment:
Maybe we should change the name of this property to Exe, to be more in line with the other properties here: deps and config. Also, we won't confuse it with the Executable class.